apis/controller/v1alpha1/devworkspaceoperatorconfig_types.go

```diff
@@ -83,9 +84,16 @@ type WorkspaceConfig struct {
 	// +kubebuilder:validation:Pattern=^[a-z0-9]([-a-z0-9]*[a-z0-9])?$
 	// +kubebuilder:validation:MaxLength=63
 	PVCName string `json:"pvcName,omitempty"`
-	// StorageClassName defines and optional storageClass to use for persistent
+	// StorageClassName defines an optional storageClass to use for persistent
 	// volume claims created to support DevWorkspaces
 	StorageClassName *string `json:"storageClassName,omitempty"`
+	// StorageClassSize defines an optional map of sizes of persistent volume claims for each storage
+	// class used by DevWorkspaces.
+	// The available keys to the map are: "common", "per-workspace".
+	// Note that the "async" storage class also uses the PVC size set for the "common" storage class.
+	// If not specified, the storage class size defaults are:
+	// 10Gi for "common" (and "async"), 3Gi for "per-workspace".
+	StorageClassSize map[string]resource.Quantity `json:"StorageClassSize,omitempty"`
```

Comment:
Json keys should not be capitalized
```suggestion
	StorageClassSize map[string]resource.Quantity `json:"storageClassSize,omitempty"`
```
Also, given future plans around making these sizes configurable in each DevWorkspace (i.e. start looking at the `size` field in Volume components, maybe something like 
```suggestion
	DefaultStorageSize map[string]resource.Quantity `json:"defaultStorageSize,omitempty"`
```
is more suitable.